db/migrate/20180710202259_add_disable_support_form_to_admin_settings.rb

```diff
@@ -0,0 +1,7 @@
+class AddDisableSupportFormToAdminSettings < ActiveRecord::Migration[5.1]
```

Comment:
There's a sliiiight chance the timestamp on this might give us some issues because we've run newer migrations. If it does, though, we know where to find you. 😉